src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -499,4 +516,57 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
     private Class loadBindClass() {
         loadClass("${MODEL_PACKAGE}.Bind")
     }
+    
+    private def parseDevice(String deviceString) {
+        List<String> tokens = deviceString.tokenize(':')
+        int numberOfTokens = tokens.size()
```

Comment:
The variable is not used anymore and tokens.size() also reads well, so I've removed the variable.